circle.yml

```diff
@@ -8,8 +11,5 @@ test:
     - mkdir -p $CIRCLE_TEST_REPORTS/junit/
     - find . -type f -regex ".*/target/.*-reports/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
     - cp -r target/coverage-reports/jacoco/ $CIRCLE_ARTIFACTS
-    - curl http://www.jpm4j.org/install/script > jpmInstall.sh
-    - sudo sh jpmInstall.sh
-    - sudo jpm install com.codacy:codacy-coverage-reporter:assembly
-    - codacy-coverage-reporter -l Java -r target/coverage-reports/jacoco/jacoco.xml --projectToken $CODACY_PROJECT_TOKEN
-    - mvn coveralls:report -DrepoToken=$COVERALLS_REPO_TOKEN 
+    - if [ ! -z $CODACY_PROJECT_TOKEN ]; then codacy-coverage-reporter -l Java -r target/coverage-reports/jacoco/jacoco.xml --projectToken $CODACY_PROJECT_TOKEN; fi
```

Comment:
See also here: https://discuss.circleci.com/t/git-errors-failing-the-build-when-deploying-to-heroku/776/4